@shared/api/tsconfig.json

```diff
@@ -3,5 +3,5 @@
   "compilerOptions": {
     "baseUrl": "."
   },
-  "include": ["."]
+  "include": [".", "../helpers/stellar.ts"]
```

Comment:
Looks like this was auto-added because I include this helper in the api layer usage now.